src/Console/Command/Compile.php

```diff
@@ -545,6 +547,10 @@ private function configureMetadata(Configuration $config, Box $box, CompilerLogg
                 'Setting metadata'
             );
 
+            if (is_file($metadata) && is_readable($metadata)) {
+                $metadata = require $metadata;
+            }
+
             if (is_callable($metadata)) {
```

Comment:
To avoid confusion where the path is also a callable function, how about using `elseif` ?
```suggestion
            } elseif (is_callable($metadata)) {
```

Reply:
Why not, but the current implementation allow also to chain it if the file included return a string (that identify a callable) rather then an array or any data structure